sqldelight-gradle-plugin/src/test/integration-postgresql/src/test/kotlin/app/cash/sqldelight/postgresql/integration/PostgreSqlTest.kt

```diff
@@ -181,6 +182,7 @@ class PostgreSqlTest {
       )
   }
 
+  @Ignore
```

Comment:
I'll take this out when I'm done testing. This test fails for me locally for some reason

Reply:
Yes, this is excepted if you don't live in UTC. Just ignore it locally until someone fix it.

Reply:
Can now be deleted, right?

```suggestion

```

Reply:
Huh, thought I took that out already. Thanks!